great_expectations/data_asset/data_asset.py

```diff
@@ -343,7 +343,9 @@ def wrapper(self, *args, **kwargs):
         return outer_wrapper
 
     def _initialize_expectations(
-        self, expectation_suite=None, expectation_suite_name=None
+        self,
+        expectation_suite: Union[Dict, ExpectationSuite, None] = None,
```

Comment:
Could this be a list too?

Reply:
I'm not familiar with `NoneType`.
My cursory google didn't find anything, do you have a reference to how it should be used?

>`Optional[X]` is equivalent to `X | None` (or `Union[X, None]`)

https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
On a second attempt, I found some mentions of `NoneType`, which is the same as `type(None)`.
But for type hints, it's common practice just to use the `None` object itself.

https://docs.python.org/3/library/types.html#types.NoneType
https://peps.python.org/pep-0484/#using-none

If you see something that suggests we should be using `NoneType` instead of `None` I'm all ears.